### DISCUSSION TITLE
Can't collapse object property / array element

### DISCUSSION THREAD

**Discussion by emohamed** [spatie/ray#218] [General]
When I click on the collapse triangle, it doesn't actually collapse the branch of the object / array:

https://user-images.githubusercontent.com/536840/105142768-a5d28980-5b03-11eb-8d4f-55f406b49adf.mp4

Not sure if this is intended, but Symfony `dump()` does have this feature.

**Comment by freekmurze** [comment]
Normally, this should work.

Could you please create an issue for this? Make sure to specific all the versions you are using.
